script/releases/v0.1.0-rewardsv2/upgrade.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "rewards-v2",
+    "from": "^0.0.1",
+    "to": "0.0.1",
```

Comment:
Should this be `0.1.0`?

Reply:
Rewards release semver will be `0.5.1` (our latest was `0.4.3`)